jupyterhub/services/service.py

```diff
@@ -160,6 +160,11 @@ class Service(LoggingConfigurable):
     def managed(self):
         """Am I managed by the Hub?"""
         return bool(self.command)
+    
+    @property
+    def kind(self):
+        """Am I managed (as text)"""
```

Comment:
Or perhaps better still: `"""Determine kind of service. True, managed; False, external."""`